src/App.js

```diff
@@ -1,15 +1,30 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
 import TodoList from './TodoList'
 import AddTodoForm from './AddTodoForm'
 
+function useSemiPersistentState(key, initialState) {
+	const savedValue = localStorage.getItem(key) || initialState
```

Comment:
Great job getting the value here! 

Since we're only using `savedValue` once, how can we incorporate this line of code into `useState`?

It's good practice to have your state be the first thing you declare in your component.